index.html

```diff
@@ -11,229 +11,236 @@
-                <ul class="media">
-                  <li>JavaScript</li>
-                  <li>Ruby</li>
-                  <li>HTML</li>
-                  <li>CSS</li>
-                </ul>
-              </ul>              
-            </div>
+    <h3>About Me</h3>
+    <p class="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    <button class="button-text about-me-button" type="button">Get my resume</button>
+    <div class="plane-top"></div>
+    <div class="plane-top-desktop"></div>
+    <div class="plane-bottom"></div>
+    <div class="plane-bottom-desktop"></div>
+    <div class="components">
+      <ul class="components-list">
+        <li>
+          <div class="categories-list">
+            <img src="./images/ellipse.svg" alt="Languages Icon">
+            <h4>Languages</h4>
+            <ul>
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+              <li id="invisible">HTML</li>
+            </ul>
           </div>
-        </article>
-        <section class="qoute">
-          <div class="holder-text">
-            <img src="images/holder.svg" alt="holder">
-            <div class="holder-text1">
-              <img src="images/ellipse2.svg" alt="holder">
-              <ul>
-                <li class="Language">
-                  <p>Frameworks</p>
-                </li>
-                <ul class="media">
-                  <li>React.js</li>
-                  <li>Ruby on Rails</li>
-                  <li>RSpec</li>
-                  <li>Capybara</li>
-                  <li>Selenium</li>
-                </ul>
-              </ul>                   
-            </div>
+        </li>
+        <li>
+          <div class="categories-list">
+            <img src="./images/ellipse2.svg" alt="Frameworks Icon">
+            <h4>Frameworks</h4>
+            <ul>
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
           </div>
-        </section>
-        <article class="qoute">
-          <div class="holder-text">
-          <img src="images/holder.svg" alt="holder">
-            <div class="holder-text1">
-              <img src="images/skills.svg" alt="holder">
-              <ul>
-                <li class="Language">
-                  <p>Skills</p>
-                </li>
-                <ul class="media">                                                  
-                  <li>Database Management</li>
-                  <li>Version Controle</li>
-                  <li>CLI</li>
-                  <li>WEb Development</li>
-                  <li>API Design</li>
-                <ul>
-              </ul> 
-            </div>
+        </li>
```

Comment:
Great job on improving your About Me section! However, a few new issues have emerged that need attention:

- [ ] Each card should have a left and right margin of 24px, but currently, it is set to 0 in your code. Please update your code to match the design requirements.
- [ ] Please ensure that you use proper padding and spacing for lists. Check the design and adjust your code accordingly.
- [ ] The gap between skill cards does not match the design. Please update the gap between the skill cards to align with the design.

![Frame 38 (1)](https://github.com/essiewanjiru99/essie-portfolio/assets/55840999/514549c6-3c3d-42b9-bcf1-b3c3f2001adb)

Keep up the good work and continue refining your project! 👍